main.c

```diff
@@ -1035,9 +1037,12 @@ static void kbd_key(void *data, struct wl_keyboard *k, uint32_t serial,
 		b = b->next;
 	}
 
-	if (!action)
-		tsm_vte_handle_keyboard(term.vte, sym, XKB_KEY_NoSymbol,
+	if (!action){
```

Comment:
Missing space, there's a space before `{` everywhere else, I'd like it to be consistent